tests/acceptance/usage-in-acceptance-test.js

```diff
@@ -0,0 +1,33 @@
+import { test } from 'qunit';
+import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+import { visit, click, fillIn, waitUntil } from 'ember-native-dom-helpers/test-support/helpers';
```

Comment:
Yeah I know we're masking intentionally but I generally think it's not the best example to show off something. I get that we may need to do that. I wonder if a comment that says "Hey we know we're masking the global helpers here, the are still available, e.g use window.find"